<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealTimeSignal</class>
 <widget class="QMainWindow" name="RealTimeSignal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>信号显示</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../Res/res.qrc">
    <normaloff>:/img/FDPC.png</normaloff>:/img/FDPC.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QCustomPlot" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>691</width>
        <height>341</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QToolBox" name="toolBox">
           <property name="currentIndex">
            <number>3</number>
           </property>
           <widget class="QWidget" name="page">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>-61</y>
              <width>325</width>
              <height>281</height>
             </rect>
            </property>
            <attribute name="label">
             <string>接收数据相关参数设置</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>数据类型：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>显示点数(个)：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>偏移量(位)：</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>绘图间隔:</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QComboBox" name="comboBox">
                   <item>
                    <property name="text">
                     <string>float</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>short</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>long</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>int</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>char *</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>double</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_offset"/>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_speed"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>131</height>
                </size>
               </property>
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QRadioButton" name="radioButton_imaginarypart">
                  <property name="text">
                   <string>复数</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButton_realpart">
                  <property name="text">
                   <string>实数</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QRadioButton" name="radioButton_sidebyside">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>并列排列</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButton_alternate">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>交替排列</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QRadioButton" name="radioButton_drawmodel">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>描绘模</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButton_drawimagine">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>描绘虚部</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButton_drawreal">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>描绘实部</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_3">
            <attribute name="label">
             <string>当前模式</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QRadioButton" name="radioButton_test">
                  <property name="text">
                   <string>测试</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButton_use">
                  <property name="text">
                   <string>实际使用</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>346</width>
              <height>220</height>
             </rect>
            </property>
            <attribute name="label">
             <string>可选项</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QGroupBox" name="groupBox_7">
               <property name="title">
                <string/>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QRadioButton" name="radioButton_datasave">
                  <property name="text">
                   <string>存储数据</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../../../Res/res.qrc">
                    <normaloff>:/img/save-pro.png</normaloff>:/img/save-pro.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_4">
            <attribute name="label">
             <string>显示范围设置</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QGroupBox" name="groupBox_6">
               <property name="title">
                <string/>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QRadioButton" name="radioButton_choose">
                  <property name="text">
                   <string>选择</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../../../Res/res.qrc">
                    <normaloff>:/img/search_hover.png</normaloff>:/img/search_hover.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButton_drag">
                  <property name="text">
                   <string>拖动</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../../../Res/res.qrc">
                    <normaloff>:/img/full-screen.png</normaloff>:/img/full-screen.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButton">
                  <property name="text">
                   <string>恢复</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../../../Res/res.qrc">
                    <normaloff>:/img/ref.jpg</normaloff>:/img/ref.jpg</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButton_run">
           <property name="text">
            <string>运行</string>
           </property>
           <property name="icon">
            <iconset resource="../../../Res/res.qrc">
             <normaloff>:/img/start.png</normaloff>:/img/start.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_pause">
           <property name="text">
            <string>暂停</string>
           </property>
           <property name="icon">
            <iconset resource="../../../Res/res.qrc">
             <normaloff>:/img/pause.jpg</normaloff>:/img/pause.jpg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_stop">
           <property name="text">
            <string>停止</string>
           </property>
           <property name="icon">
            <iconset resource="../../../Res/res.qrc">
             <normaloff>:/img/close-pro.png</normaloff>:/img/close-pro.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action">
   <property name="text">
    <string>参数设置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Res/res.qrc"/>
 </resources>
 <connections/>
</ui>
